.ci/Jenkinsfile

```diff
@@ -61,6 +61,13 @@ pipeline {
             }
           }
         }
+        withGithubNotify(context: "Unused exports") {
+          withNodeJSEnv() {
+            dir("${BASE_DIR}"){
+              sh(label: 'npm unused-exports', script: 'npm run unused-exports')
+            }
+          }
+        }
```

Comment:
Can we run `npm run unused-exports` in pre-commit? What I expect here is that we will need to wait until the Jenkins job to fail and then compose another commit fixing it.

Reply:
Yes that's an improvement I wanted to add, I just didn't have time to get it tested and working when I pushed this. I'll address this later today if I can.

Reply:
@kyungeunni can you please test that this is working now? I attempted to add it in https://github.com/elastic/synthetics-recorder/pull/189/commits/ad230e07b3fbe4d5f842dbd735b1b90387f995ac, and it looks like it works for me!

Reply:
We've discussed this offline but I'll respond here as well for posterity. We have decided that yes, we do want to run this on CI as well. It is possible to skip pre-commit hooks locally. If we don't perform a complete lint of the codebase as part of our integration process, it's possible that unused exports will be missed. And since the pre-commit hook uses `lint-staged`, we are never performing a full local check of the codebase, so when unused exports are merged they will never be caught, and we'll be back in the same situation in which we find ourselves.

By keeping this in CI, even if a contributor uploads an unused export, it will still be caught by the build machine, and we can avoid merging it as part of code review.